Formula/pillow.rb

```diff
@@ -70,6 +63,14 @@ def install
     inreplace "setup.py", "DEBUG = False", "DEBUG = True"
 
     pythons.each do |python|
+      post_args = %W[
+        --prefix=#{prefix}
+        --install-scripts=#{bin}
+        --install-lib=#{prefix/Language::Python.site_packages(python)}
+        --single-version-externally-managed
+        --record=installed.txt
+      ]
+
       system python, "setup.py", "build_ext", *pre_args, "install", *post_args
```

Comment:
```suggestion
      system python, "setup.py", "build_ext", *pre_args, "install", *post_args, "--install-lib=#{prefix/Language::Python.site_packages(python)}"
```
If you do this instead of moving `post_args` it would make the diff less noisy.

Reply:
Makes sense!
Btw, does the same`--prefix=#{prefix}` and `--install-scripts=#{bin}` for different pythons affect it in some wrong way?

Reply:
Not sure, but the need for this additional flag is due to changes in Python 3.10, which no longer infers `--install-lib` from `--prefix`. Not sure why we have `--install-scripts`.

Reply:
Okay, let's try to drop `--prefix` and `--install-scripts` then